src/components/IncidentDetails/AddDefinitionOfDone/AddDefinitionOfDoneStepper.tsx

```diff
@@ -0,0 +1,208 @@
+import { useReducer } from "react";
+import useUpdateEvidenceDOD from "../../../api/query-hooks/mutations/useUpdateEvidenceDOD";
+import { Evidence, EvidenceType } from "../../../api/services/evidence";
+import { DoDEvidenceScript } from "./steps/DoDEvidenceScript";
```

Comment:
From a naming perspective, once inside a folder, we shouldn't be reusing that Folder name or variation of it in a name. 

```suggestion
import { ScriptStep } from "./steps/DoDEvidenceScript";
```